backend/tests/Integration/Ai/AiRequestLifecycleTest.php

```
@@ -0,0 +1,309 @@
+namespace Tests\Integration\Ai;
+
+use App\Application\Ai\AiFeatureUnavailable;
+use App\Infrastructure\Ai\AiFileCleanupProcessor;
+use App\Infrastructure\Ai\DatabaseAiFileCleanupQueue;
+use App\Infrastructure\Clock;
+use App\Infrastructure\Http\AiIdempotencyStore;
+use App\Infrastructure\Http\IdempotencyConflict;
+use PHPUnit\Framework\TestCase;
+use yii\db\Connection;
+
+final class AiRequestLifecycleTest extends TestCase
```

PHPStan (2.2.7)

[error] Class Tests\Integration\Ai\AiRequestLifecycleTest extends unknown class PHPUnit\Framework\TestCase.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)